.github/workflows/build.yml

```diff
@@ -281,16 +255,9 @@ jobs:
           - ubuntu-latest
         python-version:
           - "3.7"
-          # Disabled due to an unresolvable dependency issue between sslyze and
-          # nassl:
-          # ERROR: Cannot install pshtt because these package versions have
-          # conflicting dependencies.
-          # The conflict is caused by:
-          #   sslyze 2.1.4 depends on nassl<2.3.0 and >=2.2.0
-          #   sslyze 2.1.3 depends on nassl<2.3.0 and >=2.2.0
-          # - "3.8"
-          # - "3.9"
-          # - "3.10"
+          - "3.8"
+          - "3.9"
+          - "3.10"
           # - "3.11"
         include:
```

Comment:
Please remove the matrix entry for Python 3.6 here since this PR changes support to Python 3.7+.